guides/common/modules/snip_supported-client-operating-systems.adoc

```diff
@@ -7,6 +7,11 @@ ifndef::orcharhino[]
 endif::[]
 ifdef::satellite[]
 * {EL} 7 and 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+ifdef::managing-hosts[]
+* For the purpose of xref:converting-a-host-to-rhel_{context}[conversion to RHEL]:
```

Comment:
```suggestion
* For the purpose of a xref:converting-a-host-to-rhel_{context}[conversion to RHEL]:
```

Reply:
Thank you, but we decided for a different wording :)